lib/routes.js

```diff
@@ -20,62 +20,31 @@ var conferenceID = 0;
 
 module.exports = function(app){
 
-  // Process an inbound call from an inbound
-  // call made to one of the two numbers
+  // Process an inbound call from an inbound call made to one of the two numbers
   // we've set up
-  app.get('/answer_inbound', function(req, res) {
-    // increment the conference ID so
-    // that every call has a unique conference
-    conferenceID++;
+  app.get('/answer', function(req, res) {
+    var answer_url = 'http://'+process.env['DOMAIN']+'/on-answer?conference_id='+conferenceID
```

Comment:
conferenceID is alwasy going to be 0 here

Reply:
might want to change it to req.query.conference_id

Reply:
I don't actually need it, the previous version used a conference and this one just connects a call. Well spotted!